tests/bfs/color_bsd.sh

```diff
@@ -0,0 +1 @@
+LSCOLORS="eB" bfs_diff -s rainbow -color
```

Comment:
```suggestion
LSCOLORS="eB" bfs_diff rainbow -color
```

`bfs_diff` already sorts before diffing